src/community/geopkg/src/main/java/org/geoserver/geopkg/wps/SemanticAnnotationsExtension.java

```diff
@@ -50,7 +50,8 @@ public GeoPkgExtension getExtension(String name, GeoPackage geoPackage) {
         }
 
         @Override
-        public GeoPkgExtension getExtension(Class extensionClass, GeoPackage geoPackage) {
+        public GeoPkgExtension getExtension(
+                @SuppressWarnings("rawtypes") Class extensionClass, GeoPackage geoPackage) {
```

Comment:
Same as above

Reply:
Amended to return `SemanticAnnotationsExtension`